packages/flutter/lib/src/cupertino/dialog.dart

```diff
@@ -1234,6 +1235,11 @@ class CupertinoActionSheetAction extends StatefulWidget {
   /// Destructive buttons have red text.
   final bool isDestructiveAction;
 
+  /// The cursor that will be shown when hovering over the button.
+  ///
+  /// If not passed will default to [MouseCursor.defer] and on web will default to [SystemMouseCursors.click].
```

Comment:
```suggestion
  /// Defaults to [MouseCursor.defer], except on web, where this defaults to [SystemMouseCursors.click].
```
'Default' makes 'If not passed' redundant.

Reply:
marking as resolved taken care in most recent PR